slotting/src/Cardano/Slotting/Block.hs

```diff
@@ -9,7 +9,8 @@ module Cardano.Slotting.Block
 where
 
 import Cardano.Binary (FromCBOR (..), ToCBOR (..))
-import Cardano.Prelude (NoUnexpectedThunks)
+import Cardano.Prelude (NoUnexpectedThunks, GShowK1 (..),
```

Comment:
Why is it necessary to import `GShowK1 (..)`? It would be nice if we could avoid that.

Reply:
Yes, it's an implementation detail of the `GHC.Generics` function, which relies on `coerce`, which needs the constructor "in scope". :(

I _think_ importing the module qualified would satisfy that restriction without having to list `GShowK1` explicitly. Would this be acceptable?

```
import Cardano.Prelude (NoUnexpectedThunks)
import qualified Cardano.Prelude as CPrelude
```